Fixes error in Articulation where default_joint_stiffness and default_joint_damping is not correctly set if actuator is instance of ImplicitActuator (#1580)
isaac-sim/IsaacLab: source/extensions/omni.isaac.lab/docs/CHANGELOG.rst

Changelog
---------

0.30.2 (2024-12-22)
~~~~~~~~~~~~~~~~~~~

Fixed
^^^^^

> * Fixed the issue in :class:`omni.isaac.lab.assets.Articulation` where the field
> default_joint_stiffness and default_joint_damping are not correctly set if actuator
> is instance of ImplicitActuator

Reviewer: ```suggestion
* Fixed populating default_joint_stiffness and default_joint_damping values for ImplicitActuator instances
in :class:`omni.isaac.lab.assets.Articulation` 
```
maybe something like this for more clarity.
